sdk-setup/src/mb2

```diff
@@ -2703,7 +2712,7 @@ init_patch_wrapper() (
         local untracked_added=()
         readarray -t untracked_added < <(join -v 2 \
             <(printf '%s' "$untracked_old") <(printf '%s' "$untracked_now"))
-        if [[ $untracked_added ]]; then
+        if [[ ${#untracked_added[*]} -gt 0 ]]; then
```

Comment:
How is this related to -o nounset?

Reply:
The original code fails with empty array (or more exactly an array with item at index 0 unset).